pages/agent/v3/securing.md.erb

```diff
@@ -102,6 +106,14 @@ do
 done
 ```
 
+## Signing pipelines  
+
+<section class="Docs__troubleshooting-note">
+  <p>Please remember that for maximum security it’s not recommended to use plugins.</p>
+</section>
```

Comment:
I _think_ we can remove this? we're pretty clear about the limitations already? What do you think?

Reply:
We're clear if we assume that the document is read top to bottom. If this piece is the starting point picked out from thee TOC... Better safe than sorry?

Reply:
Agreed that you can't assume that folks read top to bottom. 

But at the same time, saying "here are all the plugins you can use, but don't use them" is kinda...

Reply:
Especially as we're saying this is a more secure way to use plugins. With that in mind I think we can remove the note. What do you think?

Reply:
I gave this some thought, and yes, although we can't expect or assume that anyone is going to read top to bottom, it's better to remove the note.